src/setup.rs

```diff
@@ -42,6 +43,16 @@ pub fn run_device_setup(device: Option<Device>, device_name: &str) -> Result<()>
         }
     }
 
+    if let Some(ref wb_dev) = device.writeback_dev {
+        let writeback_path = device_sysfs_path.join("backing_dev");
+        fs::write(&writeback_path, wb_dev.as_os_str().as_bytes()).with_context(|| {
+            format!(
+                "Failed to configure write-back device into {}",
+                writeback_path.display()
```

Comment:
I'm worried about this part… As I mentioned in the discussion on #122, the user can specify a path like `/dev/disk/by-uuid/abcd-1234` and I'm not sure if the kernel will know that this is the same as `/dev/sda5`.

Unfortunately this feature requires `CONFIG_ZRAM_WRITEBACK`, and Fedora kernels are compiled without that. So I'll need to first recompile the kernel to test this :(

Reply:
Yes, and I tried that (well, I used a /dev/zvol/ path, but these also symlinks to ../zdN), and it Just Works (the file's opened with `backing_dev = filp_open(file_name, O_RDWR|O_LARGEFILE, 0)`, which literally just decays to what userspace open(2) does).
On read-back, `backing_dev` returns the real `/dev/zdN` path, but that does not matter at all.

Cf. transcript:
```
# zfs create -sV 40G tarta-zoot/test-wb
# modprobe zram
# tee /sys/block/zram0/backing_dev
/dev/zram/tarta-zoot/test-wb
/dev/zram/tarta-zoot/test-wb
tee: /sys/block/zram0/backing_dev: No such file or directory
^C
# tee /sys/block/zram0/backing_dev
/dev/zvol/tarta-zoot/test-wb
/dev/zvol/tarta-zoot/test-wb
$ cat /sys/block/zram0/backing_dev
/dev/zd176
```